### DISCUSSION TITLE
Question: Does immich detect duplicate photos?

### DISCUSSION THREAD

**Discussion by sandreas** [immich-app/immich#3663] [Q&A] [answered]
Hey, 

I did scan the FAQ and the docs, but I did not find a mention of how `immich` handles already existing pictures.

Scenario:
- I have multiple old backups only for my photos
- I would like to get rid of them
- I would like to ensure, that all photos are at one place, but no duplicates

Does `immich` take care of this? If so, how? (hash, metadata comparison, etc..)

**Comment by alextran1502** [comment, marked as answer]
Hello, Immich uses file's content (hash) for deduplication

**Reply by sandreas** [reply]
Thanks for the quick reply, that's what I wanted to know :-)
